spec/controllers/polls_controller_spec.rb

```diff
@@ -99,12 +99,23 @@
   describe 'POST #create' do
     context 'when valid attributes' do
       let(:poll_params) { FactoryGirl.attributes_for(:valid_poll) }
-      context 'and poll saving' do
-        it 'should be succes' do
-          expect(xhr :post, :create, poll: poll_params).to be_succes
+
+      context 'and user logged in' do
+        subject { xhr :post, :create, poll: poll_params }
```

Comment:
I think this subject should go right inside the  `describe 'POST #create' do` block — it will cleaner semantically, reduce the required nested context, and also will allow you to redo the test on the line 118 as `is { is_expected.to be_success }`